code/game/objects/items/weapons/AI_modules.dm

```diff
@@ -89,6 +91,23 @@ AI MODULES
 
 /obj/item/aiModule/proc/addAdditionalLaws(var/mob/living/silicon/ai/target, var/mob/sender)
 
+/mob/living/silicon/ai/proc/law_location_check(var/obj/machinery/computer/aiupload/comp, var/obj/item/aiModule/A)
+	var/list/turfs = new/list()
+	var/found_turf = FALSE
+	for(var/turf/T in range(get_turf(comp), (max(1,(16 - comp.uses - A.uses)))))
```

Comment:
One too many brackets I think. It's after a comma so it's fine without them. *(also spacing)*
```suggestion
	for(var/turf/T in range(get_turf(comp), max(1, (16 - comp.uses - A.uses))))
```